### PR TITLE
Use environment markers to only build wheels on certain platforms

### PR DESCRIPTION
**Pull request by JayH5**
https://www.python.org/dev/peps/pep-0496/

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+psycopg2==2.6.2; platform_python_implementation=='CPython'
 psycopg2cffi==2.7.4
 pycrypto==2.6.1
 python_axolotl_curve25519==0.1
 PyYAML==3.11
 rcssmin==1.0.6
 rjsmin==1.0.12
 simplejson==3.8.2
-Twisted==16.3.0
+Twisted==16.3.0; python_version<'3'
```
Comment (jerith) [inline requirements.txt:21]: Why don't we build this on py3?

Comment (jerith) [inline requirements.txt:21]: Oh, I see the commit message. Is there any real reason _not_ to build it on py3? If so, can you add a comment to that effect?

**Comment by jerith** [comment]
I think the environment markers are obscure enough that it makes sense to document them in the README. (Also, I think the README is a bit out of date.)

File: requirements.txt (hunk 1 above)
Comment (JayH5) [inline requirements.txt:21]: Mostly just that these builds are already quite slow.

**Comment by jerith** [comment]
:+1:

**Pull request merged**